config.sh

```diff
@@ -43,8 +43,8 @@ LISK_SCRIPTS_DIR="lisk-scripts-$LISK_SCRIPTS_VERSION"
 LISK_SCRIPTS_FILE="$LISK_SCRIPTS_DIR.tar.gz"
 LISK_SCRIPTS_URL="https://github.com/LiskHQ/lisk-scripts/archive/v$LISK_SCRIPTS_VERSION.tar.gz"
 
-NODE_VERSION="6.14.1"
-NODE_SHA256SUM="bdd6b8e33a6f54383ca14c15c05dbe76af7866e9fabf3c88f1cfea8d1b97286e"
+NODE_VERSION="8.12.0"
+NODE_SHA256SUM="29a20479cd1e3a03396a4e74a1784ccdd1cf2f96928b56f6ffa4c8dae40c88f2"
```

Comment:
`3df19b748ee2b6dfe3a03448ebc6186a3a86aeab557018d77a0f7f3314594ef6`

Reply:
```suggestion
NODE_SHA256SUM="29a20479cd1e3a03396a4e74a1784ccdd1cf2f96928b56f6ffa4c8dae40c88f2"
```
That's the checksum for the tar.gz file but we're using the tar.xz one.